cmd/agent/daemon/enrichment/enrichers.go

```diff
@@ -63,37 +68,45 @@ func (enricher *fileHashEnricher) Enrich(ctx context.Context, req *EnrichedConta
 		return
 	}
 
+	metricLabels := metricLabelsFromContainer(req.EbpfEvent.Container)
```

Comment:
This metric will create a high cardinality output as it includes each pod. Also it's quite few extra allocations for each exec event. Would be better to keep it without extra labels.